src/app/pages/send-tokens/components/send-tokens-confirm-drawer/send-tokens-confirm-drawer.tsx

```diff
@@ -40,22 +41,27 @@ export function SendTokensSoftwareConfirmDrawer(props: SendTokensSoftwareConfirm
       onClose={onClose}
       pauseOnClickOutside={showEditNonce}
     >
-      <Stack pb="extra-loose" px="loose" spacing="loose">
+      <Stack pb="extra-loose" px="loose" spacing="base">
         <SendTokensConfirmDetails
           amount={values.amount}
```

Comment:
I would argue here we need to use the values from the generated unsigned tx not the form values. The form values are used to generate the unsigned tx but there are conditionals present. I attempted to change these but there were errors telling me that `amount` did not exist on `unsignedTransaction.payload` which I don't understand bc that should exist on `StacksTransaction`? Any thoughts?

Reply:
Yeah makes sense. If we can use the current unsigned tx as a source of truth, let's